http/headers/Permissions-Policy.json

```diff
@@ -318,6 +318,41 @@
             }
           }
         },
+        "cross-origin-isolated": {
+          "__compat": {
+            "mdn_url": "https://developer.mozilla.org/docs/Web/API/crossOriginIsolated",
```

Comment:
This MDN URL doesn't look quite right.  Let's remove the MDN URL until a proper page has been created for this header value.